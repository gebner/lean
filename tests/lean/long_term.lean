def long_term :=
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1) =
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)+
(1+1+1+1+1+1+1+1+1)

#print long_term
