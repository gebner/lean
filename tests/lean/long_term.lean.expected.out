def long_term : Prop :=
1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) + (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
          (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
        (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
      (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
    (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) =
  1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) + (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
            (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
          (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
        (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
      (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1) +
    (1 + 1 + 1 + 1 + 1 + 1 + 1 + 1 + 1)
